### DISCUSSION TITLE
main screen black and constant crashing

### DISCUSSION THREAD

**Discussion by Turtel1** [ppy/osu#17410] [Q&A] [answered]
When i loaded up lazer or the first time i changed the res to borderless, that promptly crashed my game and i tried to reinstall many times. i can only get the game to load up by spamming space, and even then the game is loaded up and the screen in black, any suggestion on what to do?

**Comment by bdach** [comment, marked as answer]
the first suggestion would be to [attach logs](https://github.com/ppy/osu/discussions/15188)

**Reply by Turtel1** [reply]
i looked and i could change the resolution to fullscreen by changing the logs, so thank you!
